pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1188,14 +1224,109 @@ private void TestPropertySheet()
             OkDialog(dlg, () => dlg.Close());
         }
 
+        private void TestLibraryMatchPropertySheet()
+        {
+            var isSmallMolecules = AsSmallMoleculesTestMode != RefinementSettings.ConvertToSmallMoleculesMode.none;
+
+            var expectedPropertiesJson =

```

Comment:
I am still left feeling that in our code this should be a Dictionary and we should use NewtonSoft.Json which we already use elsewhere to serialize it to text.

https://www.newtonsoft.com/json/help/html/serializedictionary.htm

Reply:
Another option would be to use a single string with an insertion point for IdFileName that is either empty or the .perc.xml depending on small molecule status. But, I think I still like the dictionary option better with that value added to the dictionary only if it is not small molecules.

Reply:
I can change it to Dictionary, no problem, but I'm not sure about string insertion. IdFileName is not a single difference between the two tests. It's different precursor and different spectrum count.